src/QmlControls/ParameterEditorController.cc

```diff
@@ -495,8 +516,9 @@ void ParameterEditorController::_performSearch(void)
         _searchParameters.beginReset();
         _searchParameters.clear();
 
-        for (const QString &paraName: _parameterMgr->parameterNames(_vehicle->defaultComponentId())) {
-            Fact* fact = _parameterMgr->getParameter(_vehicle->defaultComponentId(), paraName);
+        for(const uint8_t components:_parameterMgr->componentIds())
+          for (const QString &paraName: _parameterMgr->parameterNames(components)) {
```

Comment:
And here. And, use the curly bracers after first `for`. It's very unreadable without them